assets/Steam/achievements/pack-for-web.sh

```diff
@@ -10,6 +10,6 @@ for i in $ROOTDIR/dist/icons/achievements/*.svg; do
   echo $i
   # Make background transparent and replace green with black
   # The icons will be recolored by css filters matching the player's theme
-  sed -i "s/fill:#000000;/fill-opacity: 0%;/g" "$i"
-  sed -i "s/fill:#00ff00;/fill:#000000;/g" "$i"
+  sed -i "" "s/fill:#000000;/fill-opacity: 0%;/g" "$i"
+  sed -i "" "s/fill:#00ff00;/fill:#000000;/g" "$i"
```

Comment:
It's important that there be no space between `-i` and the "".
```suggestion
  sed -i"" "s/fill:#000000;/fill-opacity: 0%;/g" "$i"
  sed -i"" "s/fill:#00ff00;/fill:#000000;/g" "$i"
```